### PR TITLE
spacewalk-proxy: Enterprise Linux modifications

### PR DESCRIPTION
**Pull request by sbluhm**
## What does this PR change?

Multiple changes to spacewalk-proxy and spacewalk-proxy-installer packages to build on Enterprise Linux 8.

- Python modifications to pass pylint.
- Spec file adaptions and improvements.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered
Part of build process. Pylint tests on RHEL based systems.

spacewalk-proxy build tested ok on
 opensuse-leap-15.3-x86_64 

spacewalk-proxy-installer build tested ok  with PR #4039 (running pylint) for
 epel-8-x86_64 
 opensuse-leap-15.3-x86_64 

- [X] **DONE**

## Links

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by jordimassaguerpla — APPROVED** [review]

**Comment by juliogonzalez** [comment]
Same as the other PR. Adding the merge-candidate label.

**Comment by juliogonzalez** [comment]
Ah no, not really. This one requires a review from @vzhestkov as well. Removing label.

**Review by vzhestkov — APPROVED** [review]
Looks good

File: proxy/proxy/rhnAuthCacheClient.py [hunk 1]
```diff
@@ -18,7 +18,9 @@
 # in this software or its documentation.
 #-------------------------------------------------------------------------------
 
-## language imports
+# Module new has been removed in future Python versions. This needs to be adapted.
+import new # pylint: disable=import-error
```
Comment (vzhestkov) [inline proxy/proxy/rhnAuthCacheClient.py:22]: @sbluhm 
This line causing error 500 on the Proxy Server as there is no such module.

Comment (sbluhm) [reply proxy/proxy/rhnAuthCacheClient.py:22]: This has been moved upwards from line 170.

**Pull request merged**